packages/expo-localization/CHANGELOG.md

```diff
@@ -9,6 +9,7 @@
 ### 🎉 New features
 
 ### 🐛 Bug fixes
+- Exception in HostObject::get for prop 'NativeUnimoduleProxy': java.lang.NullPointerException ([#16316](https://github.com/expo/expo/pull/16316) by [@nomi9995](https://github.com/nomi9995))
```

Comment:
```suggestion

- Exception in HostObject::get for prop 'NativeUnimoduleProxy': java.lang.NullPointerException ([#16316](https://github.com/expo/expo/pull/16316) by [@nomi9995](https://github.com/nomi9995))
```

Reply:
@lukmccall done

Reply:
Thanks, when the CI passes, I'll merge it ;)